eZ/Publish/API/Repository/Tests/URLAliasServiceTest.php

```diff
@@ -572,6 +573,80 @@ public function testListLocationAliases()
         return array($loadedAliases, $location);
     }
 
+    /**
+     * Test for the listLocationAliases() method.
+     *
+     * @see \eZ\Publish\API\Repository\URLAliasService::listLocationAliases()
+     */
+    public function testListLocationAliasesWithTranslations()
```

Comment:
In this case rather `...WithPrioritizedLanguagesList`. We already have `testListLocationAliasesWithLanguageCodeFilter`, so `WithTranslation` is a little bit ambiguous here.